prophet-ressa/Cargo.toml

```diff
@@ -6,4 +6,6 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-source-code-parser = { git = "https://github.com/cloudhubs/source-code-parser", rev = "2153814" }
\ No newline at end of file
+source-code-parser = { git = "https://github.com/cloudhubs/source-code-parser", rev = "b30be50" }
```

Comment:
The latest commit is `1e17268`

Reply:
Should we just always have it pull the latest version instead?

Reply:
That might be a good idea, honestly. The only problem I can think of would be if we introduce a breaking change, people don't get a choice of whether to use it or not, so they can't necessarily smoothly transition their changes.